storage/persistent-storage/persistent-storage-gce.adoc

```diff
@@ -0,0 +1,44 @@
+[id="persistent-storage-using-gce"]
+= Persistent storage using GCE Persistent Disk
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-gce
+
+toc::[]
+
+{product-title} supports GCE Persistent Disk volumes (gcePD).
+You can provision your {product-title} cluster with persistent storage
+using GCE.
+Some familiarity with Kubernetes and GCE is assumed.
+
+The Kubernetes persistent volume framework allows administrators to
+provision a cluster with persistent storage and gives users a way to
+request those resources without having any knowledge of the underlying
+infrastructure.
+
+GCE Persistent Disk volumes can be provisioned dynamically.
+
+Persistent volumes are not bound to a single project or namespace;
+they can be shared across the {product-title} cluster.
+Persistent volume claims are specific to a project or namespace and can be
+requested by users.
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying
```

Comment:
"High availability" here shouldn't be hyphenated.

Reply:
Removed the hyphen in this word.